src/crimson/osd/object_context_loader.cc

```diff
@@ -18,7 +18,7 @@ using crimson::common::local_conf;
     assert(obc->is_head());
     obc->append_to(obc_set_accessing);
     return obc->with_lock<State, IOInterruptCondition>(
-      [existed=existed, obc=obc, func=std::move(func), this] {
```

Comment:
I think is intentional @tchaikov, this is because structured bindings don't declare variables, see https://www.reddit.com/r/cpp_questions/comments/e1ralf/why_structured_bindings_cant_declare_variables/